.travis.yml

```diff
@@ -1,10 +1,5 @@
 language: python
 python:
-  - "2.6"
   - "2.7"
-  - "3.2"
-  - "3.3"
-  - "3.4"
-  - "3.5"
 # command to run tests
-script: python source/tool/test_util.py
+script: python -m unittest discover tests/software/
```

Comment:
This is the correct command, you need to modify the PYTHON_PATH to include source/tools